core/src/main/java/hudson/EnvVars.java

```diff
@@ -120,6 +120,7 @@ public EnvVars(@Nonnull Map<String,String> m) {
         }
     }
 
+    @SuppressWarnings("CopyConstructorMissesField")
```

Comment:
This is the missed field:
https://github.com/jenkinsci/jenkins/blob/811583eb6007ffa22f81c312d983816e5f36a670/core/src/main/java/hudson/EnvVars.java#L88

Because this copy constructor:
https://github.com/jenkinsci/jenkins/blob/811583eb6007ffa22f81c312d983816e5f36a670/core/src/main/java/hudson/EnvVars.java#L111
discriminates:
https://github.com/jenkinsci/jenkins/blob/811583eb6007ffa22f81c312d983816e5f36a670/core/src/main/java/hudson/EnvVars.java#L117
and only occasionally fills out that field:
https://github.com/jenkinsci/jenkins/blob/811583eb6007ffa22f81c312d983816e5f36a670/core/src/main/java/hudson/EnvVars.java#L119

Reply:
```suggestion
    @SuppressWarnings("CopyConstructorMissesField") // does not set #platform, see its Javadoc
```